src/app/containers/MostRead/Canonical/index.test.jsx

```diff
@@ -0,0 +1,104 @@
+import newsMostReadData from '#data/news/mostRead';
+import zhongwenSimpMostReadData from '#data/zhongwen/mostRead/simp';
+import { service as newsConfig } from '#app/lib/config/services/news';
+import { service as zhongwenConfig } from '#app/lib/config/services/zhongwen';
+import {
+  setStalePromoTimestamp,
+  setFreshPromoTimestamp,
+  renderMostReadContainer,
+} from '../utilities/testHelper';
+
+let container;
+
+const services = {
+  news: {
+    variant: null,
+    data: newsMostReadData,
+    config: newsConfig.default,
+    expectedLastUpdated: 'Last updated:  11 January 1970',
+  },
+  zhongwen: {
+    variant: 'simp',
+    data: zhongwenSimpMostReadData,
+    config: zhongwenConfig.simp,
+    expectedLastUpdated: '最近更新： 1970年1月11日',
+  },
+};
+
+describe('MostReadContainerCanonical', () => {
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    container = null;
+    fetch.resetMocks();
+  });
+
+  Object.keys(services).forEach(service => {
+    it(`should not render most read when lastRecordTimeStamp is not fresh for ${service}`, async () => {
+      const { variant, data: mostReadData } = services[service];
+
+      fetch.mockResponse(JSON.stringify(mostReadData));
+
+      await renderMostReadContainer({
+        container,
+        isAmp: false,
+        service,
+        variant,
+        mostReadToggle: true,
+      });
+
+      expect(container.innerHTML).toEqual('');
+    });
+
+    it(`should not render items with timestamps older than 60 days for ${service}`, async () => {
```

Comment:
*newer not older

Reply:
Upon further inspection (after a tad more sleep), this test sets each item to have a fresh date which means that the items shouldn't show a timestamp at all as the articles are newer than 60 days. Changed it from 'should not' to 'should'.